nekoyume/Assets/_Scripts/UI/Model/RecipeModel.cs

```diff
@@ -95,21 +95,25 @@ private void LoadConsumable(IEnumerable<RecipeGroup> groups)
 
             foreach (var group in groups)
             {
-                foreach (var recipeId in group.RecipeIds)
+                var key = group.Key;
+                if (!ConsumableRecipeMap.TryGetValue(key, out var model))
                 {
-                    var recipe = consumableRecipeSheet[recipeId];
-                    var key = group.Key;
+                    var name = L10nManager.Localize($"ITEM_GROUPNAME_{key}");
+                    model = new RecipeRow.Model(name, group.Grade)
+                    {
+                        ItemSubType = ItemSubType.Food,
+                    };
+                    ConsumableRecipeMap[key] = model;
+                }
 
-                    if (!ConsumableRecipeMap.TryGetValue(key, out var model))
+                foreach (var recipeId in group.RecipeIds)
+                {
+                    if (!consumableRecipeSheet.TryGetValue(recipeId, out var recipe))
                     {
-                        var name = L10nManager.Localize($"ITEM_GROUPNAME_{group.Key}");
-                        model = new RecipeRow.Model(name, group.Grade)
-                        {
-                            ItemSubType = ItemSubType.Food,
-                            StatType = recipe.GetUniqueStat().StatType
-                        };
-                        ConsumableRecipeMap[key] = model;
+                        continue;
                     }
+
+                    model.StatType = recipe.GetUniqueStat().StatType;
```

Comment:
Run the `foreach (var recipeId in group.RecipeIds)` loop, the `model.StatType` keeps being overwritten. Is this okay?

Reply:
It is okay, but it would better assign it once with `group.RecipeIds.First()`. Gonna fix it!